apps/web/config/config.exs

```diff
@@ -9,9 +9,15 @@ use Mix.Config
 config :web,
   namespace: Web,
   level_label_mapping: %{
-    "beginner" => ["Kind:Beginner", "Kind:Starter", "level:starter"],
-    "intermediate" => ["Kind:Intermediate", "level:intermediate"],
-    "advanced" => ["Kind:Advanced", "level:advanced"]
+    "starter" => ["kind:beginner", "kind:starter", "level:starter"],
```

Comment:
@ybur-yug / @jeffweiss I'm leaning towards making these a collection of strings _or_ patterns, much like Credo does in it's config: https://github.com/elixirschool/extracurricular/blob/master/.credo.exs#L25

What do you think?  I can create an issue for it if you think it's a worthwhile enhancement.

Reply:
I had some questions about how the labels are supposed to work and maybe that's because the README isn't really fleshed out.

Here's how I _assume_ extracurricular is _supposed_ to work:
 * Repo/project owners sign up their project
 * A webhook from project to extracurricular is registered (either manually w/ instructions or we do it automatically via 3 legged oauth)
 * Issue submitted in repo
 * webhook notifies extracurricular
 * we determine difficulty level
 * we add it to the website and/or tweet about it

As far as I can tell, in order for projects to work w/ extracurricular, we require them to use our label set. Is that correct?

I suppose we could automatically create those labels for folks as part of the onboarding process if we're using 3-legged oauth.

Reply:
> maybe that's because the README isn't really fleshed out

I do need to dedicate some time this week to expanding on the README to include the project goals and a basic setup of expectations for the MVP.

There's couple different ways we envision someone getting a project on the platform, keeping in mind this is an MVP and things like account sign-up are out-of-scope:

+ A project decides they want to on-board and use the webhook.  We create their account and provide the webhook URL including their API token.
+ A project agrees to participate but _will not_ add the webhook — there's any number of reasons for this.  In this case we will setup the project and rely on a bot to check for new issues on a reoccurring basis.

At some point if there's enough adoption perhaps we can force the webhook as the only means of integration.  Right now we're weighing those requirements against having data to bring people to the site and the bot is a lower barrier to entry for projects.

There is an issue (#5) to discuss automagically setting up webhooks and configuration but I think that's far off, we need to see that people want to use the app first.

Now for labels...

The ultimate goal will be to encourage projects to use a standard set of labels that are easy to grok, the same set that this project and elixir-lang uses.  That said we recognize that not everyone is going to be open to using those labels verbatim so we need to support some amount of mapping.  An example would be Phoenix which is a great addition to the site but uses a variation of the label format.

Make sense?